platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/cache/VirtualNodeCache.java

```diff
@@ -164,19 +160,11 @@ private static final class ClassVersion {
      */
     private static final Executor CLEANING_POOL = Boolean.getBoolean("syncCleaningPool")
             ? Runnable::run
-            : new ThreadPoolExecutor(
-                    CACHE_CLEANER_THREAD_COUNT,
-                    CACHE_CLEANER_THREAD_COUNT,
-                    60L,
-                    TimeUnit.SECONDS,
-                    new LinkedBlockingQueue<>(),
-                    new ThreadConfiguration(getStaticThreadManager())
-                            .setThreadGroup(new ThreadGroup("virtual-cache-cleaners"))
-                            .setComponent("virtual-map")
-                            .setThreadName("cache-cleaner")
-                            .setExceptionHandler(
-                                    (t, ex) -> logger.error("Failed to purge unneeded key/mutationList pairs", ex))
-                            .buildFactory());
+            : getStaticThreadManager()
+                    .createFixedThreadPool(
+                            "virtual-map: cache-cleaner",
+                            CACHE_CLEANER_THREAD_COUNT,
+                            (t, ex) -> logger.error("Failed to purge unneeded key/mutationList pairs", ex));
```

Comment:
What kind of work queue will be used by default? It used to be specified explicitly, now not.

Reply:
I've updated the API to allow a user supplied work queue to be specified. I will go back through the code and ensure that previously defined work queues are carried forward in this change.